README.md

```diff
@@ -66,9 +66,15 @@ People
 * [Stephen Bongner](people/stephen-bongner.md) - sb2483 - [http://ec2-13-59-91-251.us-east-2.compute.amazonaws.com:8080](http://ec2-13-59-91-251.us-east-2.compute.amazonaws.com:8080)[![CircleCI](https://circleci.com/gh/sb2483/skeleton.svg?style=svg)](https://circleci.com/gh/sb2483/skeleton)
 * [Cameron Boroumand](people/cameron-boroumand.md) - cb596 - [http://cameronboroumand.com.s3-website.us-east-2.amazonaws.com/grading.html](http://cameronboroumand.com.s3-website.us-east-2.amazonaws.com/grading.html)
 * [Mikaela Brown](people/mikaela-brown.md) - mb2587 - [https://s3.amazonaws.com/aws-website-mb-u6ygo/grading.html](https://s3.amazonaws.com/aws-website-mb-u6ygo/grading.html)
+<<<<<<< HEAD
```

Comment:
These are conflict files. You must remove these. You can check out [thes](https://help.github.com/articles/resolving-a-merge-conflict-using-the-command-line/) for how to resolve conflicts in git.